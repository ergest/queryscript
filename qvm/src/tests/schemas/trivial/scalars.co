let foo = 1;
SELECT foo;
