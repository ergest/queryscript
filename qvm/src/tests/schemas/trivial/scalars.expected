{
    "decls": {
        "let foo": Int64,
    },
    "queries": [
        TypedValue {
            type_: List(
                Record(
                    [
                        Field {
                            name: "foo",
                            type_: Atom(
                                Int64,
                            ),
                            nullable: true,
                        },
                    ],
                ),
            ),
            value: Relation(
                [
                    RecordBatch {
                        schema: Schema {
                            fields: [
                                Field {
                                    name: "foo",
                                    data_type: Int64,
                                    nullable: false,
                                    dict_id: 0,
                                    dict_is_ordered: false,
                                    metadata: None,
                                },
                            ],
                            metadata: {},
                        },
                        columns: [
                            PrimitiveArray<Int64>
                            [
                              1,
                            ],
                        ],
                        row_count: 1,
                    },
                ],
            ),
        },
    ],
}